frontend/test/pytest/test_mitigation.py

```diff
@@ -18,15 +18,17 @@
 import numpy as np
 import pennylane as qml
 import pytest
+from pennylane.transforms import exponential_extrapolate
 
 import catalyst
 from catalyst.api_extensions.error_mitigation import polynomial_extrapolation
```

Comment:
Did you figure out if it makes sense to create a similar wrapper for `exponential_extrapolation` as well?

Reply:
Needs to be resolved before merging @natestemen

Reply:
Happy to if catalyst team wants, but otherwise looks like it's not needed.